mp/game/momentum/base.fgd

```diff
@@ -1423,10 +1423,10 @@
         1 : "Yes"
     ]    
 
-    solidbsp(choices) : "Solid BSP" : 0 : "Set this if this brush is in heirarchy with a moving object of some kind, and the player can stand on this brush." =
+    solidbsp(choices) : "Solid Type" : 1 =
     [
-        0 : "No"
-        1 : "Yes"
+        0 : "VPhysics"
+        1 : "BSP"
```

Comment:
Is this exactly the same thing? "No" was changed to be "VPhysics" ? Does this mean you can't set something to not solid anymore?

Reply:
By default, the entity uses VPhysics and `solidbsp` makes it use BSP instead, so I just renamed for clarity.